### ISSUE TITLE
Inkscape 1.0rc1: Unknown option --without-gui

### ISSUE THREAD

**Issue by hroncok** [jupyter/nbconvert#1246]
In Fedora, we have Inkscape 1.0rc1, thta doesn't recognize the `--without-gui` option:

```
$ inkscape --version
Inkscape 1.0rc1 (09960d6f05, 2020-04-09)
    Pango version: 1.44.7
$ inkscape --without-gui
Unknown option --without-gui
```

nbocnvert tries to use this option.

Convert a notebook with SVG to PDF. Will try to get back with full example.

**Nbconvert version:** master

**Comment by hroncok** [comment]
```
$ pytest -k svg2pdf
============================= test session starts ==============================
platform linux -- Python 3.8.2, pytest-5.4.1, py-1.8.0, pluggy-0.13.1
rootdir: /home/churchyard/Dokumenty/jupyter/nbconvert
plugins: cov-2.6.1
collected 253 items / 248 deselected / 5 selected

nbconvert/preprocessors/tests/test_svg2pdf.py ....F                      [100%]

=================================== FAILURES ===================================
___________________________ Testsvg2pdf.test_output ____________________________

self = <nbconvert.preprocessors.tests.test_svg2pdf.Testsvg2pdf testMethod=test_output>

    @onlyif_cmds_exist('inkscape')
    def test_output(self):
        """Test the output of the SVG2PDFPreprocessor"""
        nb = self.build_notebook()
        res = self.build_resources()
        preprocessor = self.build_preprocessor()
>       nb, res = preprocessor(nb, res)

nbconvert/preprocessors/tests/test_svg2pdf.py:73: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
nbconvert/preprocessors/base.py:47: in __call__
    return self.preprocess(nb, resources)
nbconvert/preprocessors/base.py:69: in preprocess
    nb.cells[index], resources = self.preprocess_cell(cell, resources, index)
nbconvert/preprocessors/convertfigures.py:45: in preprocess_cell
    output.data[self.to_format] = self.convert_figure(
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <nbconvert.preprocessors.svg2pdf.SVG2PDFPreprocessor object at 0x7fc3f6c0e5e0>
data_format = 'image/svg+xml'
data = '<?xml version="1.0" encoding="UTF-8" standalone="no"?>\n<!-- Created with Inkscape (http://www.inkscape.org/) -->\n<s...;stroke-dasharray:none;stroke-dashoffset:0.00000000;stroke-opacity:1.0000000"\n       id="rect5719" />\n  </g>\n</svg>'

    def convert_figure(self, data_format, data):
        """
        Convert a single SVG figure to PDF.  Returns converted data.
        """
    
        # Work in a temporary directory
        with TemporaryDirectory() as tmpdir:
    
            # Write fig to temp file
            input_filename = os.path.join(tmpdir, 'figure.svg')
            # SVG data is unicode text
            with io.open(input_filename, 'w', encoding='utf8') as f:
                f.write(cast_unicode_py2(data))
    
            # Call conversion application
            output_filename = os.path.join(tmpdir, 'figure.pdf')
            shell = self.command.format(from_filename=input_filename,
                                   to_filename=output_filename)
            subprocess.call(shell, shell=True) # Shell=True okay since input is trusted.
    
            # Read output from drive
            # return value expects a filename
            if os.path.isfile(output_filename):
                with open(output_filename, 'rb') as f:
                    # PDF is a nb supported binary, data type, so base64 encode.
                    return base64.encodebytes(f.read())
            else:
>               raise TypeError("Inkscape svg to pdf conversion failed")
E               TypeError: Inkscape svg to pdf conversion failed

nbconvert/preprocessors/svg2pdf.py:136: TypeError
----------------------------- Captured stderr call -----------------------------
Unknown option --without-gui
=============================== warnings summary ===============================
__venv38__/lib64/python3.8/site-packages/nbformat/notebooknode.py:4
  /home/churchyard/Dokumenty/jupyter/nbconvert/__venv38__/lib64/python3.8/site-packages/nbformat/notebooknode.py:4: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
    from collections import Mapping

__venv38__/lib64/python3.8/site-packages/jinja2/utils.py:485
  /home/churchyard/Dokumenty/jupyter/nbconvert/__venv38__/lib64/python3.8/site-packages/jinja2/utils.py:485: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
    from collections import MutableMapping

nbconvert/preprocessors/tests/test_svg2pdf.py::Testsvg2pdf::test_inkscape_pre_v1_command
  /home/churchyard/Dokumenty/jupyter/nbconvert/nbconvert/preprocessors/tests/test_svg2pdf.py:90: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(preprocessor.command, 'fake-inkscape --without-gui --export-pdf="{to_filename}" "{from_filename}"')

nbconvert/preprocessors/tests/test_svg2pdf.py::Testsvg2pdf::test_inkscape_v1_command
  /home/churchyard/Dokumenty/jupyter/nbconvert/nbconvert/preprocessors/tests/test_svg2pdf.py:94: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(preprocessor.command, 'fake-inkscape --without-gui --export-file="{to_filename}" "{from_filename}"')

nbconvert/preprocessors/tests/test_svg2pdf.py::Testsvg2pdf::test_inkscape_version_default
  /home/churchyard/Dokumenty/jupyter/nbconvert/nbconvert/preprocessors/tests/test_svg2pdf.py:82: DeprecationWarning: Please use assertEqual instead.
    self.assertEquals(preprocessor.inkscape_version, '0.92.3')

-- Docs: https://docs.pytest.org/en/latest/warnings.html
=========================== short test summary info ============================
FAILED nbconvert/preprocessors/tests/test_svg2pdf.py::Testsvg2pdf::test_output
=========== 1 failed, 4 passed, 248 deselected, 5 warnings in 0.72s ============
```

**Comment by hroncok** [comment]
There is also: `Unknown option --export-file=/tmp/tmp4xj1exbv/figure.pdf` later if I get rid of `--without-gui`

**Comment by hroncok** [comment]
```
Usage:
  org.inkscape.Inkscape [OPTION…] file1 [file2 [fileN]]

Process (or open) one or more files.

Help Options:
  -?, --help                                 Show help options
  --help-all                                 Show all help options
  --help-gapplication                        Show GApplication options
  --help-gtk                                 Show GTK+ Options

Application Options:
  -V, --version                              Print Inkscape version
  --system-data-directory                    Print system data directory
  --user-data-directory                      Print user data directory
  -p, --pipe                                 Read input file from standard input (stdin)
  --pdf-page=PAGE                            PDF page number to import
  --pdf-poppler                              Use poppler when importing via commandline
  --convert-dpi-method=[...]                 Method used to convert pre-0.92 document dpi, if needed: [none|scale-viewbox|scale-document]
  --no-convert-text-baseline-spacing         Do not fix pre-0.92 document's text baseline spacing on opening
  -o, --export-filename=EXPORT-FILENAME      Output file name (file type is guessed from extension)
  --export-overwrite                         Overwrite input file
  --export-type=[...]                        File type(s) to export: [svg,png,ps,eps,pdf,emf,wmf,xaml]
  -C, --export-area-page                     Area to export is page
  -D, --export-area-drawing                  Area to export is whole drawing (ignoring page size)
  -a, --export-area=x0:y0:x1:y1              Area to export in SVG user units
  --export-area-snap                         Snap the bitmap export area outwards to the nearest integer values
  -d, --export-dpi=DPI                       Resolution for bitmaps and rasterized filters; default is 96
  -w, --export-width=WIDTH                   Bitmap width in pixels (overrides --export-dpi)
  -h, --export-height=HEIGHT                 Bitmap height in pixels (overrides --export-dpi)
  --export-margin=MARGIN                     Margin around export area: units of page size for SVG, mm for PS/EPS/PDF
  -i, --export-id=OBJECT-ID[;OBJECT-ID]*     ID(s) of object(s) to export
  -j, --export-id-only                       Hide all objects except object with ID selected by export-id
  -l, --export-plain-svg                     Remove Inkscape-specific SVG attributes/properties
  --export-ps-level=PS-Level                 Postscript level (2 or 3); default is 3
  --export-pdf-version=PDF-VERSION           PDF version (1.4 or 1.5)
  -T, --export-text-to-path                  Convert text to paths (PS/EPS/PDF/SVG)
  --export-latex                             Export text separately to LaTeX file (PS/EPS/PDF)
  --export-ignore-filters                    Render objects without filters instead of rasterizing (PS/EPS/PDF)
  -t, --export-use-hints                     Use stored filename and DPI hints when exporting object selected by --export-id
  -b, --export-background=COLOR              Background color for exported bitmaps (any SVG color string)
  -y, --export-background-opacity=VALUE      Background opacity for exported bitmaps (0.0 to 1.0, or 1 to 255)
  -I, --query-id=OBJECT-ID[,OBJECT-ID]*      ID(s) of object(s) to be queried
  -S, --query-all                            Print bounding boxes of all objects
  -X, --query-x                              X coordinate of drawing or object (if specified by --query-id)
  -Y, --query-y                              Y coordinate of drawing or object (if specified by --query-id)
  -W, --query-width                          Width of drawing or object (if specified by --query-id)
  -H, --query-height                         Height of drawing or object (if specified by --query-id)
  --vacuum-defs                              Remove unused definitions from the <defs> section(s) of document
  --select=OBJECT-ID[,OBJECT-ID]*            Select objects: comma-separated list of IDs
  --actions=ACTION(:ARG)[;ACTION(:ARG)]*     List of actions (with optional arguments) to execute
  --action-list                              List all available actions
  --verb=VERB[;VERB]*                        List of verbs to execute
  --verb-list                                List all available verbs
  -g, --with-gui                             With graphical user interface (required by some actions/verbs)
  --batch-process                            Close GUI after executing all actions/verbs
  --shell                                    Start Inkscape in interactive shell mode
  --display=DISPLAY                          X display to use


Examples:
  Export input SVG (in.svg) to PDF (out.pdf) format:
	inkscape --export-filename=out.pdf in.svg
  Export input files (in1.svg, in2.svg) to PNG format keeping original name (in1.png, in2.png):
	inkscape --export-type=png in1.svg in2.svg
  See 'man inkscape' and http://wiki.inkscape.org/wiki/index.php/Using_the_Command_Line for more details.
```

**Comment by MSeal** [comment]
Thanks for reporting and fixing!
